grakn-graph/src/main/java/ai/grakn/graph/internal/computer/GraknSparkComputer.java

```diff
@@ -86,37 +96,24 @@
 
     private final org.apache.commons.configuration.Configuration sparkConfiguration;
     private boolean workersSet = false;
+    private final ThreadFactory threadFactoryBoss =
+            new BasicThreadFactory.Builder().namingPattern(GraknSparkComputer.class.getSimpleName() + "-boss").build();
 
-    private static GraknGraphRDD graknGraphRDD = null;
+    private final ExecutorService computerService = Executors.newSingleThreadExecutor(threadFactoryBoss);
 
-    private org.apache.commons.configuration.Configuration apacheConfiguration = null;
-    private Configuration hadoopConfiguration = null;
+    static {
+        TraversalStrategies.GlobalCache.registerStrategies(GraknSparkComputer.class,
+                TraversalStrategies.GlobalCache.getStrategies(GraphComputer.class).clone().addStrategies(
+                        SparkSingleIterationStrategy.instance(),
+                        SparkInterceptorStrategy.instance()));
+    }
```

Comment:
Yes. We still wanna make sure multiple compute queries can be executed at the same time.